install-dev/controllers/console/process.php

```diff
@@ -23,34 +23,39 @@
  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
+use PrestaShopBundle\Install\Install;
+use PrestaShopBundle\Install\Database;
 
-class InstallControllerConsoleProcess extends InstallControllerConsole
+class InstallControllerConsoleProcess extends InstallControllerConsole implements HttpConfigureInterface
 {
 
-    protected $model_install;
+    protected $model_database;
     public $process_steps = array();
     public $previous_button = false;
 
     public function init()
     {
-        require_once _PS_INSTALL_MODELS_PATH_.'install.php';
-        require_once _PS_INSTALL_MODELS_PATH_.'database.php';
-        $this->model_install = new InstallModelInstall();
+        $this->model_install = new Install();
         $this->model_install->setTranslator($this->translator);
 
-        $this->model_database = new InstallModelDatabase();
+        $this->model_database = new Database();
         $this->model_database->setTranslator($this->translator);
     }
 
     /**
-     * @see InstallAbstractModel::processNextStep()
+     * @see HttpConfigureInterface::processNextStep()
      */
     public function processNextStep()
     {
     }
 
+    public function display()
```

Comment:
Because it now implements HttpConfigureInterface and the function display() is mandatory